src/cheat_sheet.py

```diff
@@ -134,24 +129,21 @@ def draw(self, canvas):
         self.draw_items(canvas, actions_2)
         self.next_column(canvas)
 
+        all_scopes = get_list("scope_type")
+        scopes_limit = round(len(all_scopes) - 3)
```

Comment:
Rounding because copied from division code. -3 is show some because it's fitting with the current content. I have no better way of doing this for now.